src/fetch_sex_places.py

```diff
@@ -0,0 +1,407 @@
+import asyncio
+import json
+import math
+import os
+import re
+from argparse import ArgumentParser
+from csv import DictWriter
+from configparser import RawConfigParser
+from datetime import date
+from io import StringIO
+from itertools import chain
+from pathlib import Path
+from urllib.parse import urlencode
+
+import aiofiles
+import pandas as pd
+from aiohttp import request
+from geopy.distance import vincenty
+
+
+class SexPlacesNearBy:
+
+    BASE_URL = 'https://maps.googleapis.com/maps/api/place/'
+    KEYWORDS = ('acompanhantes',
+                'adult entertainment club',
+                'adult entertainment store',
+                'gay sauna',
+                'massagem',
+                'modeling agency',
+                'motel',
+                'night club',
+                'sex club',
+                'sex shop',
+                'strip club',
+                'swinger clubs')
+
+    def __init__(self, company, key=None):
+        """
+        :param company: (dict) Company with name, cnpj, latitude and longitude
+        :param key: (str) Google Places API key
+        """
+        settings = RawConfigParser()
+        settings.read('config.ini')
+        self.company = company
+        self.key = key or settings.get('Google', 'APIKey')
+        self.latitude = self.company['latitude']
+        self.longitude = self.company['longitude']
+        self.places = []
+        self.closest = None
+
+    @property
+    def company_name(self):
+        return self.company.get('trade_name') or self.company.get('name')
+
+    async def get_all_places(self):
+        """
+        Use aiohttp to get the closest place to the company according to each
+        keyworn. Returns `None`, the results are saved on self.places.
+        """
+        tasks = [self.load_place(k) for k in self.KEYWORDS]
+        await asyncio.gather(*tasks)
+
+    async def get_closest(self):
+        """
+        Gets the closest place find in reads self.places, queries for its
+        details and returns a dictoinary with all the info on that place.
```

Comment:
typo `dictoinary` and i would suggest using just `dict` when referencing the type.